management/prompt.go

```diff
@@ -30,3 +36,39 @@ func (m *PromptManager) Read(opts ...RequestOption) (p *Prompt, err error) {
 func (m *PromptManager) Update(p *Prompt, opts ...RequestOption) error {
 	return m.Request("PATCH", m.URI("prompts"), p, opts...)
 }
+
+// MarshalJSON is a custom serializer for the promptCustomText type.
+//
+// We have to use one to avoid creating a new Request method that does not JSON-encode the request body.
+func (t *promptCustomText) MarshalJSON() ([]byte, error) {
+	return []byte(auth0.StringValue(t.Text)), nil
+}
+
+// UnmarshalJSON is a custom deserializer for the promptCustomText type.
+//
+// We have to use one to avoid creating a new Request method that does not JSON-decode the response body.
+func (t *promptCustomText) UnmarshalJSON(b []byte) error {
+	t.Text = auth0.String(string(b))
+	return nil
+}
+
+// Retrieve custom text for a specific prompt and language.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Prompts/get_custom_text_by_language
+func (m *PromptManager) CustomText(p string, l string, opts ...RequestOption) (t *string, err error) {
+	var r *promptCustomText
+	err = m.Request("GET", m.URI("prompts", p, "custom-text", l), &r, opts...)
+	t = r.Text
+	return
+}
+
+// Set custom text for a specific prompt. Existing texts will be overwritten.
```

Comment:
Comment should have the following format 'SetCustomText ...' (with an optional leading article)

From: https://golang.org/doc/effective_go#commentary

Reply:
Changed in https://github.com/go-auth0/auth0/pull/240/commits/1016644b84f927ba9fc2741a649cee693e808dad